addon/components/variable-number/number.ts

```diff
@@ -11,14 +11,13 @@ import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 import { service } from '@ember/service';
 import intlService from 'ember-intl/services/intl';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
 import { localCopy } from 'tracked-toolbox';
 import {
   MAXIMUM_VALUE_PNODE_KEY,
   MINIMUM_VALUE_PNODE_KEY,
 } from '@lblod/ember-rdfa-editor-lblod-plugins/plugins/variable-plugin/utils/constants';
 import { isBlank } from '@ember/utils';
+import n2words from 'n2words';
```

Comment:
You can specify only importing one language ([wiki-link](https://github.com/forzagreen/n2words/wiki/Importing-only-specific-languages), but I see it gives some typescript errors, so if you also couldn't get it to work, this is fine (I feel like the problem is with the library somewhere).
But I wanted to at least mention it :)